fdmi/plugins/fdmi_plugin_st.sh

```diff
@@ -45,13 +49,13 @@ do_some_kv_operations()
 {
 	local rc=0
 
-	for ((i=0; i < 1; i++)) ; do
+	for ((i=0; i<1; i++)) ; do
 		DIX_FID="12345:12345$i"
 		MOTR_PARAM="-l ${lnet_nid}:$SNS_MOTR_CLI_EP  \
 			    -h ${lnet_nid}:$HA_EP -p $PROF_OPT \
 			    -f $M0T1FS_PROC_ID -s "
 
-		$M0_SRC_DIR/utils/m0kv ${MOTR_PARAM}                                       \
+		"$M0_SRC_DIR/utils/m0kv" ${MOTR_PARAM}                                       \
```

Comment:
We can mark it as "false positive". No need to change it.

Reply:
agreed